src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -709,11 +709,10 @@ self =>
     def checkQMarkUsage() =
       if (!settings.isScala3 && isRawIdent && in.name == raw.QMARK)
         deprecationWarning(in.offset,
-          "`?` in a type will be interpreted as a wildcard in the future, wrap it in backticks to keep the current meaning.", "2.13.6")
+          "Wrap `?` in backticks to use it as an identifier, or use `-Xsource:3` to use it as a wildcard like in Scala 3.", "2.13.6")
```

Comment:
```suggestion
          "Wrap `?` in backticks to continue to use it as an identifier, or use `-Xsource:3` to use it as a wildcard like in Scala 3.", "2.13.6")
```

I'm struggling with how exactly to phrase this (without being too verbose) but I wanted to help clarify how the code is interpreted, while neither changes are made (i.e when the warning is being emitted).